aliases/available/docker.aliases.bash

```diff
@@ -9,7 +9,14 @@ alias dkpsa='docker ps -a'  # List all Docker containers
 alias dki='docker images'  # List Docker images
 alias dkrmac='docker rm $(docker ps -a -q)'  # Delete all Docker containers
 alias dkrmlc='docker-remove-most-recent-container'  # Delete most recent (i.e., last) Docker container
-alias dkrmui='docker images -q -f dangling=true |xargs docker rmi'  # Delete all untagged Docker images
+case $OSTYPE in
+  darwin*)
```

Comment:
You should probably add `bsd*)` here, too, since macOS and BSD share the same `xargs` parameters.